keylime/ima/ima.py

```diff
@@ -212,18 +212,19 @@ def _validate_ima_ng(
             failure.add_event("not_in_allowlist", f"File not found in allowlist: {path.name}", True)
             return failure
 
-        if codecs.encode(digest.hash, "hex").decode("utf-8") not in accept_list:
+        hex_hash = digest.hash.hex()
```

Comment:
My guess is we didn't know about the hex() method on bytes and therefore used the more complicated code...